README.md

```diff
@@ -54,6 +57,8 @@ Atalhos para utilizar emojis mais facilmente no Git, CRM, Slack, etc.
 :) ou -0 → 😃  
 :D ou -D → 😄  
 -P → 😜  
+-clap → 👏  
```

Comment:
boa!
Acho que poderia ter um do foguete tb 🚀

Reply:
Eu tb sugiro, nao sei se vocês usam, o in-progress (🚧)

Reply:
concordo com fuguete! 🚀

Reply:
E não é que já tem 😄, é o **-go**, só uso por esse atalho.

Reply:
eita